.github/workflows/ci.yml

```diff
@@ -17,6 +17,6 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
       - name: Build
-        run: make -j8 V=1
+        run: make -j8 V=1 CONFIGS="shared cpp11-shared"
```

Comment:
Should this be `cpp98-shared cpp11-shared`?

Reply:
No, we call "cpp98-shared" simply "shared".

Reply:
`shared` was renamed to `cpp98-shared`